### PR TITLE
Fixing duplicate logging

### PR DESCRIPTION
**Pull request by aghodke1312**
**What type of PR is this?**
> Uncomment only one and also add the corresponding label in the PR:
>bug
>feature
improvement
>cleanup
>api-change
>design
>documentation
>failing-test
>unit-test
>integration-test

**What this PR does / why we need it**:
Reduced the duplicate logging. there were 12 log lines repeated. Reduced it to 6. 
https://jenkins.pwx.dev.purestorage.com/job/Stork/job/stork-ocp-kubevirt-4-13/171/consoleFull
Now thees 6 are because `kubectl logs stork-test  -n kube-system -f
for i in $(seq 1 100) ; do
    test_status=$(kubectl get pod stork-test -n kube-system -o json | jq ".status.phase" -r)
    if [ "$test_status" = "Running" ]; then
        echo "Test is still running, status: $test_status"
        kubectl logs stork-test  -n kube-system -f
    else
        sleep 5
        break
    fi
done` 

**Does this PR change a user-facing CRD or CLI?**:
no

**Is a release note needed?**:
no

**Does this change need to be cherry-picked to a release branch?**:
no

### CODE REVIEW

File: test/integration_test/common_test.go [hunk 1]
```diff
@@ -1939,15 +1939,14 @@ func TestMain(m *testing.M) {
 	var once sync.Once
 	once.Do(func() {
 		doDashboardSetup()
+		if err := setup(); err != nil {
+			log.Error("Setup failed with error: %v", err)
+			os.Exit(1)
+		}
+		exitCode := m.Run()
```
Comment (pureneelesh) [inline test/integration_test/common_test.go:1946]: So, is TestMain getting invoked multiple times in the same job? If yes, that seems to be a problem since that would mean the same set of tests are running multiple times?

**Review by pureneelesh — APPROVED** [review]

**Comment by pureneelesh** [comment]
@aghodke1312 : changes look good. Can you please fix the commit/PR title and description since this is now a different change from the original one?

**Comment by aghodke1312** [comment]
Running one more set of test with line causing duplicate logs removed.

**Review by Rohit-PX — APPROVED** [review]
LGTM.
